pandas/tests/frame/test_constructors.py

```diff
@@ -999,6 +1001,17 @@ def __len__(self, n):
                             array.array('i', range(10))])
         tm.assert_frame_equal(result, expected, check_dtype=False)
 
+    def test_constructor_range(self):
+        # GH26342
+        result = DataFrame(range(10))
+        expected = DataFrame(list(range(10)))
+        tm.assert_frame_equal(result, expected)
+
+    def test_constructor_list_of_ranges(self):
+        result = DataFrame([range(10), range(10)])
```

Comment:
I would reference the issue as a comment above this line as well.

Reply:
Oh, this is just an existing test that has been moved. It already passed before this PR.